docs/functions.rst

```diff
@@ -1572,6 +1572,83 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+.. _restoredatausingkopiaserver:
+
+RestoreDataUsingKopiaServer
+---------------------------
+
+This function restores data backed up by the ``BackupDataUsingKopiaServer`` function.
+It creates a new Pod that mounts the PVCs referenced by the specified Pod
+and restores data to the specified path.
+
+.. note::
+   It is extremely important that, the PVCs are not be currently
+   in use by an active application container, as they are required
+   to be mounted to the new Pod (ensure by using
+   ``ScaleWorkload`` with replicas=0 first).
+   For advanced use cases, it is possible to have concurrent access but
+   the PV needs to have ``RWX`` mode enabled and the volume needs to use a
+   clustered file system that supports concurrent access.
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `namespace`, Yes, `string`, namespace of the application that you want to restore the data in
+   `image`, Yes, `string`, image to be used for running restore job (should contain kopia binary)
+   `backupIdentifier`, Yes, `string`, unique snapshot id generated during backup
+   `restorePath`, Yes, `string`, path where data to be restored
+   `userPassphrase`, Yes, `string`, user access credentials for kopia repository server
+   `certData`, Yes, `string`, certificate data for kopia repository server
+   `pod`, No, `string`, pod to which the volumes are attached
+   `volumes`, No, `map[string]string`, Mapping of `pvcName` to `mountPath` under which the volume will be available
+   `podOverride`, No, `map[string]interface{}`, specs to override default pod specs with
+
+.. note::
+   The ``image`` argument requires the use of ``ghcr.io/kanisterio/kanister-tools``
+   image since it includes the required tools to restore data from
+   the object store.
+   Between the ``pod`` and ``volumes`` arguments, exactly one argument
```

Comment:
The RestoreData in general is being used to restore data for BackupData and CopyVolumeData. 
So in case of Restoring snapsot created by BackupData we mention pod and
In case of of restoring snapshot created by CopyVolumeData we mention volumes

Reply:
You could check it [here](http://localhost:63342/kanister/docs/_build/html/functions.html?_ijt=3156deoqoo25u4o6rk44nvfjr6&_ij_reload=RELOAD_ON_SAVE#restoredata)